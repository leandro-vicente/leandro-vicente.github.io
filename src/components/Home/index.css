.avatar {
  border-radius: 100%;
  height: 200px;
  width: 200px;
}

#contact-button{
  margin-top: 80px;
}