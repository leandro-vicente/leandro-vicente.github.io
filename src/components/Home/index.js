import React from 'react'
import Bio from '../Bio'
import SocialBox from '../SocialBox'

import { SOCIAL_ICONS_AND_LINKS } from '../../constants'

const Home = () => {
  return(
    <React.Fragment>
      <Bio name='Leandro Vicente' tagline='Software Developer and Content Creator' />
      <SocialBox iconsAndLinks={SOCIAL_ICONS_AND_LINKS} />
    </React.Fragment>
  )
}

export default Home