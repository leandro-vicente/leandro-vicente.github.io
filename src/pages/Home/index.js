import React from 'react'
import Bio from '../../components/Bio'
import './style.css'

const Home = () => 
  <Bio name="Leandro Vicente" tagline="Software Developer and Creator" />
    
export default Home 