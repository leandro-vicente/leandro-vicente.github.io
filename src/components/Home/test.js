import React from 'react'
import { shallow } from 'enzyme'
import Home from './index'

describe('Home', () => {
  it('renders home component correctly', () => {
    const home = shallow(<Home />)
    expect(home).toMatchSnapshot()
  })
})