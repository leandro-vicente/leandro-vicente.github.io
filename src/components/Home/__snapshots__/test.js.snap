// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders home component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <Bio
          name="Leandro Vicente"
          tagline="Software Developer and Content Creator"
        />,
        <SocialMediaBox
          iconsAndLinks={
            Array [
              Object {
                "name": "linkedin",
                "url": "https://www.linkedin.com/in/leandrovicentesilva/",
              },
              Object {
                "glyphicon": "github",
                "name": "github",
                "url": "https://github.com/llevvi/",
              },
              Object {
                "name": "youtube",
                "url": "http://youtube.com/levvitalk",
              },
              Object {
                "name": "instagram",
                "url": "http://instagram.com/levvitalk",
              },
            ]
          }
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "name": "Leandro Vicente",
          "tagline": "Software Developer and Content Creator",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "iconsAndLinks": Array [
            Object {
              "name": "linkedin",
              "url": "https://www.linkedin.com/in/leandrovicentesilva/",
            },
            Object {
              "glyphicon": "github",
              "name": "github",
              "url": "https://github.com/llevvi/",
            },
            Object {
              "name": "youtube",
              "url": "http://youtube.com/levvitalk",
            },
            Object {
              "name": "instagram",
              "url": "http://instagram.com/levvitalk",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <Bio
            name="Leandro Vicente"
            tagline="Software Developer and Content Creator"
          />,
          <SocialMediaBox
            iconsAndLinks={
              Array [
                Object {
                  "name": "linkedin",
                  "url": "https://www.linkedin.com/in/leandrovicentesilva/",
                },
                Object {
                  "glyphicon": "github",
                  "name": "github",
                  "url": "https://github.com/llevvi/",
                },
                Object {
                  "name": "youtube",
                  "url": "http://youtube.com/levvitalk",
                },
                Object {
                  "name": "instagram",
                  "url": "http://instagram.com/levvitalk",
                },
              ]
            }
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "name": "Leandro Vicente",
            "tagline": "Software Developer and Content Creator",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "iconsAndLinks": Array [
              Object {
                "name": "linkedin",
                "url": "https://www.linkedin.com/in/leandrovicentesilva/",
              },
              Object {
                "glyphicon": "github",
                "name": "github",
                "url": "https://github.com/llevvi/",
              },
              Object {
                "name": "youtube",
                "url": "http://youtube.com/levvitalk",
              },
              Object {
                "name": "instagram",
                "url": "http://instagram.com/levvitalk",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
